tests/evm/precompiles/test_ecAdd.py

```diff
@@ -0,0 +1,172 @@
+import pytest
+
+from typing import List
+from common import CallContext
+from zkevm_specs.ecc_circuit import EcAdd
+from zkevm_specs.evm_circuit import (
+    Bytecode,
+    CallContextFieldTag,
+    ExecutionState,
+    Precompile,
+    RWDictionary,
+    StepState,
+    Tables,
+    verify_steps,
+)
+from zkevm_specs.util import (
+    Word,
+    FQ,
+)
+from zkevm_specs.evm_circuit.table import EccOpTag, EccTableRow
+
+
+def gen_testing_data():
```

Comment:
It would be good to have testing data about point at infinity.

Reply:
added in [9997f92](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/500/commits/9997f929b03f2a1f8aa72fcfec98cc50611d13c0)